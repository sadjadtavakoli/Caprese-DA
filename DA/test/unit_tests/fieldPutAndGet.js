let obj1 = {
    name:"david",
    job: "plumber",
    talk: function(){
        let variable = this.name
    }
}

obj1.name = "berke"
obj1.talk()

// main impacts obj1.talk