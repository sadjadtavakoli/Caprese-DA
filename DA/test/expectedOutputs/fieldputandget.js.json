{"talk-test/unit_tests/fieldputandget.js-4-6":{"impacted":["t_1"]},"test/unit_tests/fieldputandget.js":{"impacted":["t_5"]},"keyMap":{"t_1":"test/unit_tests/fieldputandget.js","t_5":"talk-test/unit_tests/fieldputandget.js-4-6"}}